packages/pigeon/lib/cpp_generator.dart

```diff
@@ -588,7 +588,7 @@ const flutter::StandardMessageCodec& ${api.name}::GetCodec() {
                     indent.write('if ($encodableArgName.IsNull()) ');
                     indent.scoped('{', '}', () {
                       indent.writeln(
-                          'wrapped.emplace(flutter::EncodableValue("${Keys.error}"), WrapError("$argName unexpectedly null."));');
+                          'wrapped = WrapError("$argName unexpectedly null.");');
```

Comment:
Note for future PR work: this is something we should remove, and replace with a debug-only assert, since per previous offline discussion we don't want non-debug runtime checks that the wire format is correct. Could you see if we have a bug about removing runtime type/nullability checks in our deserialization across any generators that have them, and file it if not?

Reply:
Issue filed, feel free to add more detail if you think my description is lacking. https://github.com/flutter/flutter/issues/116972